config.toml

```diff
@@ -0,0 +1,52 @@
+baseURL = 'https://hive.apache.org/'
+languageCode = 'en-us'
+title = 'Hive Site'
+theme = 'hive'
+[params]
+  hiveLogo = "/images/hive.svg"
+  logo = "/images/asf_logo.png"
```

Comment:
Ideally, the logo should be clickable. You can hide the link 'Apache Hive' in the logo, so that it becomes clickable. You can increase the size of the logo a bit if you can.

Reply:
Done